CameraManager.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Models.ModelConfigurations;

```

Comment:
seems a duplicate file of TEnCu - Framework/Assets/Scripts/Utility/CameraManager/CameraManager.cs